trieste/utils/pareto.py

```diff
@@ -106,6 +113,164 @@ def _bounds_2d(front: TensorType) -> BoundedVolumes:
 
         return BoundedVolumes(lower, upper)
 
+    @staticmethod
+    def _is_test_required(smaller: TensorType) -> TensorType:
+
+        # Tests if a point augments or dominates the Pareto set.
+        # :param smaller: a boolean tf.Tensor storing test point < Pareto front
+        # :return: True if the test point dominates or augments the Pareto front (boolean)
+
+        # if and only if the test point is at least in one dimension smaller
+        # for every point in the Pareto set
+        idx_dom_augm = tf.reduce_any(smaller, axis=1)
+        is_dom_augm = tf.reduce_all(idx_dom_augm)
+
+        return is_dom_augm
+
+    def divide_conquer_nd(self, front: TensorType, threshold: TensorType = 0) -> TensorType:
+        # Divide and conquer strategy to compute the cells covering the non-dominated region.
+        # Generic version: works for an arbitrary number of objectives.
+
+        len_front, number_of_objectives = front.shape
+        jitter = gpflow.config.default_jitter()
```

Comment:
we don't use gpflow's jitter in trieste, so as to avoid mutable globals. Instead you can use a default argument like [in `BatchReparametrizationSampler`](https://github.com/secondmind-labs/trieste/blob/develop/trieste/acquisition/function.py#L601). I'm not sure how you'd make this make sense for the 2D case, maybe you can think of sth

Reply:
is it make sense if we only pass the jitter to divide_conquer_nd? and not passing it to 2d case

Reply:
that makes sense, but you'll need it in `__init__`. On reflection, I don't think that's so bad as long as it has a default, which it will

Reply:
DEFAULT is defined in utils, circular import occurred, seems I need to move pareto.py back outside utils

Reply:
might be easier, at least for reviewing's sake, to move DEFAULTS to misc.py and import DEFAULTS into __init__.py

Reply:
okay i'll move DEFAULTS to misc.py . should I move back Pareto to utils?

Reply:
yeah that'd be great - it's difficult to follow the PR otherwise